### PR TITLE
translate the open libery splash page

### PR DESCRIPTION
**Pull request by aknguyen7**
1. Add this in the bnd.bnd file
`Provide-Capability: welcome.translation.provider; welcome.translation.provider="OSGi-INF/welcome/js/externalize-string.js"`
2. create js/externalize-string.js to handle translate message
3. create nls/messages.js to contains translate message for both open/close liberty splash page
4. move banner.js to js/banner.js
5. update banner.js to handle the message
6. update index.html to do the translation and also fix DAP errors

### CODE REVIEW

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    EXTERNAL_RESOURCES: "External Resources",
+    WASDEV: "WASdev Community",
+    LIBERTY_DOC: "Liberty Documentation",
+    WASDEV_FORUM: "WASdev Forum",
+    HEADER_UPDATE_AVAILABLE: "Update Available: ",
+    HEADER_DOWNLOAD_LINK: "Click to download {0}", // Product name
+    HEADER_UPDATE_AVAILABLE_DOWNLOAD_LINK: "Update Available. Click to download {0}", // Product name
+    COPYRIGHT: "Copyright",
+    IBM_LOGO: "IBM Logo",
+    IBM_LICENSED: "Licensed Materials — Property of IBM © Copyright IBM Corp. 1997, 2019. All Rights Reserved. IBM, and the IBM logo are trademarks or registered trademarks of International Business Machines Corp., registered in many jurisdictions worldwide. Other product and service names might be trademarks of IBM or other companies. A current list of IBM trademarks is available on the Web at {0} ", // COPYRIGHT_URL
```
Comment (tevans78) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js]: Is it nescesary to have this IBM copyright statement in Open Liberty code? If possible it should be removed.

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    WELCOME_IBM_OPEN_SOURCE: "An IBM Open Source Project",
+    RESOURCES: "Resources",
+    GROUPS_IO_FORUM: "Groups.io forum",
+    COPYRIGHT_OPEN_LIBERTY: "Licensed Materials — Property of the Open Liberty project, Copyright IBM Corp and others 1997, 2019. Project is distributed under the terms of the Eclipse Public License v1.",
+    WELCOME: "Welcome to Liberty",
+    WEBSPHERE_LIBERTY_LOGO: "WebSphere Liberty Logo",
+    WEBSPHERE_OPEN_LIBERTY_FOOTER: "WebSphere Liberty footer Logo",
+    WELCOME_SECTION: "Welcome Section",
+    WELCOME_SECTION_CONTENT: "Welcome Section Content",
+    WEBSPHERE_APPSERVER: "WebSphere Application Server Beta",
```
Comment (tevans78) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js]: This doesn't seem applicable to Open Liberty either? Where/when is it used?

Comment (tevans78) [reply dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js]: Suggest that this file contain only those messages which are actually used by OL.

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+var messages = {
+    WELCOME_OPEN_LIBERTY: "Welcome to Open Liberty",
+    OPEN_LIBERTY_APP_SERVER: "Open Liberty Application Server",
+    WEBSPHERE_OPEN_LIBERTY_LOGO: "WebSphere Liberty Logo",
```
Comment (tevans78) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js]: and I hate to be picky but it isn't "WebSphere Open Liberty", just "Open Liberty". Please change the variable names as well as the text.

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js (hunk 2 above)
Comment (aknguyen7) [reply dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/nls/messages.js]: I updated the messages.js to contain only messages from OL. Thanks!

**Comment by aknguyen7** [comment]
for #9115

**Comment by aknguyen7** [comment]
#build

File: dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/js/banner.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+		} else {
+			createDownloadLink(section);
+			createCloseButton(section);
+		}
+		media.addListener(updateBannerTabOrder);
+
+		var welcomeSection = document.getElementById("welcome-section");
+		document.body.insertBefore(section, document.body.firstChild);
+	}
+}
```
Comment (k8vance88) [inline dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/js/banner.js]: At the end of the previous version of this file, it called buildUpdateBanner(); Why was this removed?

Comment (aknguyen7) [reply dev/com.ibm.ws.transport.http/resources/OSGI-INF/welcome/js/banner.js]: cuz i call that in the index.html to handle the translation for the banner
